tests/testthat/test-html-tweak.R

```diff
@@ -162,9 +162,20 @@ test_that("badges can't contain an extra text", {
   )
 })
 
-test_that("badges can be in special div", {
-  expect_equal(
-    badges_extract_text('<p></p><div id="badges"><a href="x"><img src="y"></a></div>'),
-    '<a href="x"><img src="y"></a>'
-  )
+test_that("badges-paragraph à la usethis can be found", {
+  string <- '
```

Comment:
I think you can indent this string?

Reply:
:heavy_check_mark:

Reply:
It needs to be indented one more level so that it's clearly inside the test block.